test/test.js

```diff
@@ -144,6 +162,20 @@ describe( "Localization", function() {
         );
     });
 
+    it( "uses the default locale if language code query parameter is not in the list", function( done ) {
```

Comment:
The default locale is needed in case no language is set at all. If a user explicitly sets the wrong language I think we should return 404.